README.md

```diff
@@ -1,62 +1,28 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
+# First Mobile Version
```

Comment:
Kindly give a proper title to your project along with a proper description. Having a proper title and description is an absolute must for any project on GitHub since this is the first thing your visitors will refer to, to get a general idea about the project. Also, kindly add other sections to your README file like how to set up the project, and prerequisites for the project. Also, consider updating the Acknowledgements section since it is a copy from the template provided by Microverse.